Add stack rocks option.

DavidBlackberry opened the pull request:
<!-- Reviewable:start -->
[<img src="https://reviewable.io/review_button.png" height=40 alt="Review on Reviewable"/>](https://reviewable.io/reviews/udacity/create-your-own-adventure/2782)
<!-- Reviewable:end -->

cbuckey-uda commented:
@casey-collab, please review this

casey-collab commented:
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

casey-collab commented on english/scarab/goddess.md:
```diff
@@ -16,4 +16,6 @@ Do you:
\ No newline at end of file
+[ask for forgiveness?](https://www.youtube.com/watch?v=fGuR4GOV5L0)
+
+[Stack rocks until the next move comes to you?](english/scarab/stack-rocks/stack-rocks.md)
```
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

casey-collab commented on english/scarab/stack-rocks/stack-rocks.md:
```diff
@@ -0,0 +1 @@
+You have chosen wisely.
```
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

DavidBlackberry commented:
Thanks! I think I've fixed it now. I guess that directory(scarab) wasn't accessible from marshmallow.md, so I've added a link to it and fixed the link I added to scarab/goddess.md.

cbuckey-uda commented:
Thanks for the update, @DavidBlackberry! @casey-collab, please take another look.

casey-collab commented:
Looks great! :dog:
